styles/styles.css

```diff
@@ -782,3 +782,125 @@ main .section.small-vertical-margins {
         text-align: left;
     }
 }
+
+.getting-started main .default-content-wrapper ul {
+    margin: 0;
+    padding: 10px;
+}
+
+.getting-started main .default-content-wrapper li {
+    padding-bottom: 10px;
+}
+
+.getting-started main .default-content-wrapper p {
+    margin: 0 0 10px;
+}
+
+.getting-started main .default-content-wrapper img {
+    transform: scale(0.6);
+}
+
+main .section[data-background] picture.section-background-img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    object-fit: cover;
+    z-index: -2;
+    overflow: hidden;
+}
+
+main .section[data-background] picture.section-background-img img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.getting-started main .default-content-wrapper p > strong:first-child:last-child {
+    font-size: 1.7rem;
+    color: var(--link-color);
+}
+
+main .section.centered p:has(picture:first-child:last-child) {
+    text-align: center;
+}
+
+.getting-started main .section.grey {
+    background-color: var(--background-grey-4);
+}
+
+.getting-started main .default-content-wrapper .button-container {
+    max-width: 180px;
```

Comment:
not sure we should impose a max-width of 180px, in some cases it will display the button text on two lines where there are two words

Reply:
also, by setting a max-width, the ::after pseudo-element is cut a bit
![image](https://github.com/hlxsites/bevespi/assets/26941572/ef0be054-d336-4deb-a7a2-41db28bff186)